Util/DoctrineHelper.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the DunglasApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dunglas\ApiBundle\Util;
```

Comment:
Should be in the `Doctrine` namespace and if possible with a more explicit name.